package.json

```diff
@@ -15,11 +15,15 @@
     "checkchange": "beachball check",
     "publish:beachball": "beachball publish",
     "start": "node ./scripts/watch.js",
-    "test": "lerna run test"
+    "test": "lerna run test",
+    "lint": "eslint packages --ext .ts,.js"
```

Comment:
Sorry for asking this after merge !
@43081j as this is monorepo ! shouldnt the lint command be like this 
```json
 "lint": "eslint packages \"packages/**/!(node_modules)/**/*.ts\" \"packages/**/!(node_modules)/**/*.js\" --ext .ts,.js"
```
**?**
what do you think ?

Reply:
No worries. 

It can either be a glob or a directory with `ext` but not both. 

The `.eslintignore` should handle ignoring node modules and what not. 

It's possible it should really be `packages/*/src/**/*.{ts,is}` but not all packages are structured the same (i.e some have no src).

Reply:
ohh, great then  😃